x-pack/plugins/uptime/public/components/synthetics/check_steps/steps_list.tsx

```diff
@@ -147,7 +147,7 @@ export const StepsList = ({ data, error, loading }: Props) => {
   };
 
   return (
-    <EuiPanel>
+    <EuiPanel hasBorder>
```

Comment:
I'd leave this without border and padding. We don't need the panel styles, but we can leave the component as a wrapper if needed.

<img width="1481" alt="Screenshot 2021-06-29 at 22 24 10" src="https://user-images.githubusercontent.com/4104278/123862915-39b54680-d929-11eb-9f77-f8525cb66190.png">